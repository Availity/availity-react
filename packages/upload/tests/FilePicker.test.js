import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { FilePicker } from '..';

afterEach(cleanup);

describe('Upload', () => {
  test('should render', () => {
    const { container } = render(<FilePicker onChange={() => {}} />);

    expect(container).toMatchSnapshot();
  });
});
