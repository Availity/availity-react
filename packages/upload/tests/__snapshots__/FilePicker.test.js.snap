// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload should render 1`] = `
<div>
  <div
    class="custom-file"
  >
    <input
      class="custom-file-input"
      id="filePicker-1"
      type="file"
      value=""
    />
    <label
      class="custom-file-label"
      for="filePicker-1"
    >
      Choose file
    </label>
  </div>
</div>
`;
